code/modules/reagents/reagent_containers/blood_pack.dm

```diff
@@ -1,23 +1,33 @@
+#define BLOODPACK_NORMAL 1
+#define BLOODPACK_CUT 2
+
 /obj/item/weapon/reagent_containers/blood
-	name = "\improper Bloodpack"
+	name = "\improper bloodpack"
 	desc = "Contains blood used for transfusion."
 	icon = 'icons/obj/bloodpack.dmi'
-	icon_state = "empty"
+	icon_state = "bloodpack"
 	volume = 200
 
 	var/blood_type = null
+	var/holes = 0
+	var/mode = BLOODPACK_NORMAL
 
 /obj/item/weapon/reagent_containers/blood/New()
 	..()
+	processing_objects.Add(src)
```

Comment:
Since the processing is only used for dripping AFTER the bloodbag is punctured, only move it to processing_objects after the puncture